runtime/lua/vim/F.lua

```diff
@@ -5,40 +5,28 @@ local F = {}
 ---@param a
 ---@param b
 function F.if_nil(a, b)
-  if a == nil then
-    return b
-  end
+  if a == nil then return b end
   return a
 end
 
 -- Use in combination with pcall
 function F.ok_or_nil(status, ...)
-  if not status then
-    return
-  end
+  if not status then return end
```

Comment:
this is counter to our C style guidelines and in Lua the visual control flow is also helpful. For cases where one wants to avoid several lines you can still use a ternary.